source/elements/oneTBB/source/algorithms/blocked_ranges/blocked_nd_range_cls.rst

```diff
@@ -45,8 +45,20 @@ For example, ``blocked_nd_range<int,2>`` is analogous but not identical to ``blo
                 // Access
                 bool is_divisible() const;
                 const dim_range_type& dim(unsigned int dimension) const;
-            };
+            }; // class blocked_nd_range
 
+            // Deduction Guides
+            template <typename Value, typename... Values>
+            blocked_nd_range(blocked_range<Value>, blocked_range<Values>...)
+            -> blocked_nd_range<Value, 1 + sizeof...(Values)>;
+
+            template <typename Value, unsigned int N>
+            blocked_nd_range(const Value (&)[N], typename blocked_nd_range<Value, N>::size_type = 1)
+            -> blocked_nd_range<Value, N>;
+
+            template </*see-below*/>
```

Comment:
I prefer keeping the previous: 

`template <typename Value>`
`blocked_nd_range(/*a set of braced-init-lists of Value objects*/)`
`        -> blocked_nd_range<Value, /*number of braced-init-lists*/>;`

And then the detailed description provides the two options for how to do that. What's currently here seems even less clear.

Reply:
You might add the "see-below" comment in addition, like `blocked_nd_range(/*a set of braced-init-lists of Value objects (see-below)*/)`

Reply:
Applied the combined approach with the comment below.